packages/babel-traverse/src/path/interface-compatibility.js

```diff
@@ -0,0 +1,77 @@
+let deprecateShown = false;
+function showDeprecationWarning() {
+  if (!deprecateShown) {
+    console.trace(
+      "Accessing NodePath.skipKeys is deprecated, use NodePath.skipKey(key) instead",
```

Comment:
I'm ok with adding this deprecation message, but I'd prefer to wait for a minor release.

Reply:
Oh I find out another way to skip the whole `skipKeysSet` things and achieve similar performance.